CryptKeeperZK/crypt-keeper-extension
# chore: optimize build

## Explanation

Get rid of clean scripts and replace source maps to get rid of eval code + improve build speed a little bit.

Details are bellow:
- [x] Remove clean scripts
- [x] Update source maps

## More Information

Fixes #107 

## Screenshots/Screencaps

N/A

## Manual Testing Steps

Just try to run `npm run dev` and `npm run build` to check if extension loads properly.

## Pre-Merge Checklist


### + If there are functional changes:


> PR template source from [github.com/MetaMask](https://github.com/MetaMask)
